src/setupTests.ts

```diff
@@ -1,7 +1,7 @@
+import 'core-js/stable';
 import { configure } from 'enzyme';
 import EnzymeReactAdapter from 'enzyme-adapter-react-16';
 import 'jest-enzyme';
-import 'babel-polyfill';
```

Comment:
Så babel gjorde split.regex biten før? Nå går alt via core-js?

Reply:
Dette er bare for test-biten, her ble babel-polyfill brukt slik at `Array.prototype.flatMap` etc skulle eksistere under test. 
Men babel-polyfill har blitt deprecated til fordel for core-js, så bytter derfor om på det i test-sammenheng også.

Reply:
Aha. Da skjønner jeg! Derfor fetch-mock også forsvinner?